cmd/influxd/run/server.go

```diff
@@ -63,7 +64,8 @@ type Server struct {
 	BindAddress string
 	Listener    net.Listener
 
-	Logger zap.Logger
+	Logger            zap.Logger
```

Comment:
Should we upgrade to a more recent version of zap? I think that this is an older version.

Reply:
This might be a change for later, but could also be useful to do now.

Reply:
I think we should be upgrading to a newer version, but I thought this change would get unwieldy if I did it right here. The change is already decently hard to track and I've had a bad track record when I try to mix moving refactors with other stuff.